specification/cognitiveservices/data-plane/AzureOpenAI/authoring/stable/2022-12-01/azureopenai.json

```diff
@@ -0,0 +1,1915 @@
+          "$ref": "#/definitions/ScaleSettings"
+        }
+      ],
+      "properties": {
+        "capacity": {
+          "format": "int32",
+          "description": "The constant reserved capacity of the inference endpoint for this deployment.",
+          "type": "integer"
+        }
+      },
+      "x-ms-discriminator-value": "manual"
+    },
+    "ManualScaleSettingsUpdate": {
+      "title": "ManualScaleSettingsUpdate",
+      "description": "Settings for the manual scale type.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ScaleSettingsUpdate"
+        }
+      ],
+      "properties": {
+        "capacity": {
+          "format": "int32",
+          "description": "The constant reserved capacity of the inference endpoint for this deployment.",
+          "type": "integer"
+        }
+      },
+      "x-ms-discriminator-value": "manual"
+    },
+    "Model": {
+      "title": "Model",
+      "description": "A model is either a base model or the result of a successful fine tune job.",
+      "required": [
+        "capabilities",
+        "deprecation",
+        "lifecycle_status"
+      ],
+      "type": "object",
+      "properties": {
+        "object": {
+          "$ref": "#/definitions/TypeDiscriminator"
+        },
+        "status": {
+          "$ref": "#/definitions/State"
+        },
+        "created_at": {
+          "format": "unixtime",
+          "description": "A timestamp when this job or item was created (in unix epochs).",
+          "type": "integer",
+          "readOnly": true
+        },
+        "updated_at": {
+          "format": "unixtime",
+          "description": "A timestamp when this job or item was modified last (in unix epochs).",
+          "type": "integer",
+          "readOnly": true
+        },
+        "id": {
+          "description": "The identity of this item.",
+          "type": "string",
+          "readOnly": true
+        },
+        "model": {
+          "description": "The base model ID if this is a fine tune model; otherwise `null`.",
+          "type": "string"
+        },
+        "fine_tune": {
+          "description": "The fine tune job ID if this is a fine tune model; otherwise `null`.",
+          "type": "string"
+        },
+        "capabilities": {
+          "$ref": "#/definitions/Capabilities"
+        },
+        "lifecycle_status": {
+          "$ref": "#/definitions/LifeCycleStatus"
+        },
+        "deprecation": {
+          "$ref": "#/definitions/Deprecation"
+        }
+      }
+    },
+    "ModelList": {
+      "title": "ModelList",
+      "description": "Represents a list of models.",
+      "type": "object",
+      "properties": {
+        "object": {
+          "$ref": "#/definitions/TypeDiscriminator"
+        },
+        "data": {
+          "description": "The list of items.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Model"
+          },
+          "readOnly": true
```

Comment:
Another `readOnly: true` you can probably drop.

I really need to add a linter rule to point these out. 😄